modules/rtm/R/invert_prospect.R

```diff
@@ -0,0 +1,29 @@
+## Default inversion
+
+invert_prospect <- function(spectra,
```

Comment:
The old documentation was no longer valid--I've always heard that wrong documentation is worse than none at all. When I get a chance, I'll sit town and fill in all the missing documentation.